cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpPlatformResources.java

```diff
@@ -396,11 +400,13 @@ public CloudVmTypes virtualMachinesForDistroX(CloudCredential cloudCredential, R
             for (Map.Entry<String, Set<VmType>> stringSetEntry : cloudVmResponses.entrySet()) {
                 returnVmResponses.put(stringSetEntry.getKey(), stringSetEntry.getValue().stream()
                         .filter(enabledDistroxInstanceTypeFilter)
+                        .filter(e -> minimalHardwareFilter.suitableAsMinimumHardware(e))
```

Comment:
Probably it is a bug, but the `virtualMachines()` call in line 396 (via `getCloudVmTypes()`) already takes care of instance filtering (for DL as well). So either the change in `getCloudVmTypes()` is not needed, or this change and the one in line 409 are not needed.

Reply:
removing virtual machine filter thanks